### PR TITLE
Fix not proper current position taken when having scroll only by horizontal/vertical linked scrollbar.

### PR DESCRIPTION
**Pull request by skrustev**

### CODE REVIEW

**Comment by skrustev** [comment]
Done :)

File: src/js/modules/infragistics.ui.scroll.js [hunk 1]
```diff
@@ -1336,8 +1336,13 @@
 				this._syncElemsX(this._container[ 0 ], false);
 				/*self._syncHBar(this._container[ 0 ], false);*/
 
-				var curY = this._getContentPositionY();
-				this._updateScrollBarsPos(destX, curY, true);
+				var curPosY;
```
Comment (mpavlinov) [inline src/js/modules/infragistics.ui.scroll.js]: Move this declaration at the top of the function.

File: src/js/modules/infragistics.ui.scroll.js [hunk 2]
```diff
@@ -1386,8 +1391,13 @@
 				this._syncElemsY(this._container[ 0 ], false);
 				/*this._syncVBar(this._container[ 0 ], false);*/
 
-				var curX = this._getContentPositionX();
-				this._updateScrollBarsPos(curX, destY, true);
+				var curPosX;
```
Comment (mpavlinov) [inline src/js/modules/infragistics.ui.scroll.js]: Move this declaration at the top of the function.

File: src/js/modules/infragistics.ui.scroll.js [hunk 3]
```diff
@@ -1992,7 +2002,19 @@
 				this._scrollFromSyncContentH = false;
 			}
 
-			this._updateScrollBarsPos(this._container.scrollLeft(), this._container.scrollTop());
+			var posX, posY;
```
Comment (mpavlinov) [inline src/js/modules/infragistics.ui.scroll.js]: Move this declaration at the top of the function.

**Comment by kdinev** [comment]
I'm approving the PR and merging it. It turns out the jQuery preset for JSCS doesn't enforce one var declaration. We will add it to our config, but this would happen at a later stage.

**Pull request merged**